lib/test-data.json

```diff
@@ -8,45 +8,21 @@
       "inputs": [
         {
           "id": "0+input-a",
-          "label": "Alpha"
-        },
-        {
-          "id": "0+input-b",
-          "label": "Beta"
+          "label": "Alpha",
+          "direction": "W",
+          "percentage": 25
         }
+        
       ],
-      "outputs": [
-        {
-          "id": "0-output-d",
-          "label": "Delta"
-        }
-      ]
+      "outputs": []
```

Comment:
Why are we getting rid of the output here? Shouldn't we test both inputs and outputs?